applications/asset_tracker/README.rst

```diff
@@ -121,33 +153,32 @@ After programming the application and all prerequisites to your board, test the
 #. Connect to the board with a terminal emulator, for example, LTE Link Monitor.
-#. Open a web browser and navigate to https://nrfcloud.com/.
-   Follow the instructions to set up your account and add an LTE device.
-   A pattern of switch and button actions is displayed.
+#. Reset the board.
 #. Observe in the terminal window that the board starts up in the Secure Partition Manager and that the application starts.
    This is indicated by output similar to the following lines::
 
       SPM: prepare to jump to Non-Secure image
       ***** Booting Zephyr OS v1.13.99 *****
       Application started
 
-#. Observe that LED 3 starts blinking as the LTE link is established. This may take several minutes.
-#. Observe in the terminal window that the connection to nRF Cloud is established.
+#. Observe in the terminal window that the connection to nRF Cloud is established. This may take several minutes.
+#. Open a web browser and navigate to https://nrfcloud.com/.
+   Follow the instructions to set up your account and add an LTE device.
 #. The first time you start the application, pair the device to your account:
 
-   a. Observe that both LED 3 and 4 start blinking, indicating that the pairing procedure has been initiated.
-   #. Follow the instructions on `nRF Cloud`_ and enter the displayed pattern.
-      In the terminal window, you can see the pattern that you have entered.
-   #. If the pattern is entered correctly, the board and your nRF Cloud account are paired and the device reboots.
-      If the LEDs start blinking in pairs, check in the terminal window which error occurred.
+   a. Observe that the LED(s) indicate that the device is waiting for user association.
+   #. Follow the instructions on `nRF Cloud`_ to pair your device.
+   #. If the pairing was successful, the board and your nRF Cloud account are paired, and the device reboots.
+      If the LED(s) indicate an error, check in the terminal window which error occurred.
       The device must be power-cycled to restart the pairing procedure.
-   #. After reboot, the board connects to the nRF Cloud, and the pattern disappears from the web page.
-#. Observe that LED 4 is turned on to indicate that the connection is established.
+   #. After reboot, the board connects to nRF Cloud.
+#. Observe that the LED(s) indicate that the connection is established.
 #. Observe that the device count on your nRF Cloud dashboard is incremented by one.
 #. Select the device from your device list on nRF Cloud, and observe that sensor data and modem information is received from the board.
-#. Toggle switch 1 to simulate flipping the board orientation.
-#. Set switch 2 in the position marked **N.C.** and observe that simulated GPS data is sent to the nRF Cloud.
-#. Optionally send AT commands from the terminal, and observe that the reponse is received.
+#. Press button 1 to send BUTTON data to nRF Cloud.
+#. Press button 1 for a minimum of 10 seconds to enable GPS tracking.
```

Comment:
Suggestion: Should we mention SW3 in the case of Thingy specifically?

Reply:
Added in parentheses.